ui/apps/platform/cypress/integration/userinfo.test.js

```diff
@@ -88,6 +89,9 @@ describe('User Profile', () => {
             cy.get('.pf-c-dropdown__menu-item:contains("Invite users")').click();
 
             checkInviteUsersModal();
+
+            // test closing the modal
+            cy.get(`${accessModalSelectors.button}:contains("Cancel")`).click();
```

Comment:
How about a helper **function** for close via cancel which tests can import?

Goal to **encapsulate selectors** as much as possible, especially outside of accessControl folder.